src/applications/registry.json

```
@@ -1214,5 +1214,15 @@
       "layout": "page-react.html",
       "title": "Connected Devices"
     }
+  },
+  {
+    "appName": "Transfer Account",
+    "entryName": "mhv-inherited-proofing",
+    "rootUrl": "/transfer-account",
+    "template": {
+      "vagovprod": false,
+      "title": "Transfer Account",
+      "layout": "page-react.html"
+    }
   }
-]
+]
```

```suggestion
]

```
Looks like the newline was removed

---

👀